### ISSUE TITLE
Suggestion: Posts to Terms

### ISSUE THREAD

**Issue by geminorum** [scribu/wp-posts-to-posts#260]
Normally we use taxonomies to assign terms to posts, but what if the taxonomy is not registered to posts. say users, say custom posts, comments, say even the other taxonomies; or in WordPress terms the objects.

Another great advantage is, the connection metadata between posts and terms. I can use term metadata table, but they just assigned to terms and not the connection between them and the posts.

So in other words and with the fortunate p2p_type column, I blissfully suggest P2P_Term_Query class.

btw, thanks again for the plugin. :)
